packages/drivers/routerlicious-driver/src/restWrapperBase.ts

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { AxiosRequestConfig, AxiosRequestHeaders } from "axios";
+
+export abstract class RestWrapper {
```

Comment:
If cloning this from server, it could be beneficial to also include the recently merged changes from #15106